website/docs/api/top-level.md

```diff
@@ -488,6 +491,47 @@ logging the results.
 
 </Project>
 
+#### spacy.CometLogger.v1 {#CometLogger tag="registered function"}
+
+> #### Installation
+>
+> ```bash
+> $ pip install comet_ml
+> ```
+
+The `CometLogger` is built into spaCy and can track all of the useful
+information about your experiment and the environment it was created
+in. This includes training and evaluation metrics and parameters, but
+also details on the Python and OS environment, output, and system usage
+metrics. You can manage running experiments, see `diffs` with other
+experiments, and share analyses through custom visualizations with
+your team at [Comet.ML](https://www.comet.ml/) . Of course, you can
+control exactly which items are logged, or not, through configuration
+settings. To create your Comet.ML API key, visit:
+[www.comet.ml/docs/python-sdk/spacy](https://www.comet.ml/docs/python-sdk/spacy/)
+
+> #### Example config
+>
+> ```ini
+> [training.logger]
+> @loggers = "spacy.CometLogger.v1"
+> project_name = "monitor_spacy_training"
+> remove_config_values = ["paths.train", "paths.dev"]
+> ```
+
+| Name                   | Description                                                                                                                           |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| `project_name`         | The name of the project in your Comet.ML workspace. The project will be created automatically if it doesn't exist yet. ~~str~~ |
+| `remove_config_values` | A list of values to remove from the config before it is logged to Comet.ML (default: empty). ~~List[str]~~                              |
+
+<Project id="integrations/comet">
+
+To quickly see how easy it is to track your spaCy training runs in
+Comet.ML try our project template. It trains on the IMDB Movie Review
+Dataset and includes a simple config with the built-in `CometLogger`.
+
+</Project>
```

Comment:
This needs to be removed - there isn't actually a project called `integrations/comet`, so this will create a dead link. I don't think we would want to have an individual project per type of logger, but f we start having more of these, perhaps we'd need to have a project that shows off all the integrations for experiment tracking though.

Reply:
I was working on that project. But ok, I'll remove it. I guess I should also remove wandb's then as well.

Reply:
Done!

Reply:
Please don't remove any of the current content - we've worked hard to get these example projects in place and to align the documentation to them. This PR should focus on getting the new logger implemented and properly documented - referring to existing projects and functionality in the docs only. If it gets merged & released, we can discuss how to potentially update the relevant example projects in a next stage.

Reply:
We can't publish a project around new functionality until the relevant new spaCy version is released, anyway, so it's not feasible to push all this in one PR.

Reply:
How about if I add the `integrations/comet` project first, and then I can refer to it? I'm just trying to keep these two logger examples aligned.

Reply:
That doesn't work from a technical point of view. The functionality needs to be in spaCy first, in a published release, until a spacy project can refer to it.

Reply:
Ok, I think I have addressed all of the concerns. Thanks for the feedback!